nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/resources/docs/org/apache/nifi/processors/standard/EncryptAttributes/additionalDetails.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
The message contained here is not relevant to `EncryptAttributes` so this file can be removed.